packages/back-end/types/datasource.d.ts

```diff
@@ -115,6 +115,12 @@ export interface UserIdType {
   description?: string;
 }
 
+export type DataSourceEvents = {
+  experimentEvent?: string;
+  experimentIdProperty?: string;
+  variationIdProperty?: string;
+};
+
```

Comment:
This nested type had no name. I wasn't sure what to call it so this is what I chose. If we have a better name, let me know and I can update it.